docs/document-parts.md

```diff
@@ -31,53 +40,32 @@ abstract: ../abstract.md
 ---
 ```
 
-### Known Frontmatter Parts
```

Comment:
Where is this getting moved to?

Reply:
We could leave this in the reference side?

Reply:
Good catch, definitely an accident!